presto-orc/src/main/java/com/facebook/presto/orc/metadata/KeyProvider.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata;
+
+public enum KeyProvider
+{
+    UNKNOWN,
```

Comment:
This does look weird. Do we want to drop it and throw an exception instead?

Reply:
see https://github.com/prestodb/presto/pull/14483#discussion_r420431194